src/WingetCreateTests/WingetCreateTests/UnitTests/GitHubTests.cs

```diff
@@ -107,5 +107,51 @@ public async Task RemoveWhitespaceFromBranchName()
             await this.gitHub.ClosePullRequest(pullRequest.Number);
             StringAssert.StartsWith(string.Format(GitHubPullRequestBaseUrl, this.WingetPkgsTestRepoOwner, this.WingetPkgsTestRepo), pullRequest.HtmlUrl, PullRequestFailedToGenerate);
         }
+
+        /// <summary>
+        /// Verifies that manifest metadata is automatically filled through GitHub's API if we have a GitHub Installer URL.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Test]
+        public async Task ValidateAutoPopulatedManifestMetadata()
+        {
+            Manifests manifests = new Manifests
+            {
+                InstallerManifest = new(),
+                DefaultLocaleManifest = new(),
+                LocaleManifests = new(),
+                VersionManifest = new(),
+            };
+
+            manifests.InstallerManifest.Installers.Add(new()
+            {
+                InstallerUrl = "https://github.com/microsoft/PowerToys/releases/download/v0.82.1/PowerToysUserSetup-0.82.1-x64.exe",
+                InstallerSha256 = "C346622DD15FECA8184D9BAEAC73B80AB96007E6B12CAAD46A055FE44A5A3908",

```

Comment:
@ryfu-msft Instead of me taking a random release from PowerToys repo, and using that to fetch the metadata, would it be better to use https://github.com/microsoft/winget-pkgs-submission-test repo for this purpose? If you agree, then we need to add the necessary metadata to that repo namely:
- Add some tags
- Generate a sample release
- Add a sample wiki

Reply:
I think that's a great idea! I've published a release here for you to use: https://github.com/microsoft/winget-pkgs-submission-test/releases/tag/v1.0.1

The release should have the binary, tags, and release notes that you can use to add an e2e test. Let me know if you require any additional information.

Reply:
@ryfu-msft Thanks! I'll also need some repo tags (for testing auto-fill of `Tags:` field) and a sample wiki (for testing auto-fill of `Documentations:` field). Screenshot from powertoys repo attached as reference:

![Screenshot 2024-08-06 123756](https://github.com/user-attachments/assets/99ad892f-cce7-44e9-a50e-5f95dfbc9a32)